src/main/resources/static/admin/AdminPanelPage.html

```diff
@@ -0,0 +1,369 @@
+        <div class="module_content">
+            <article class="stats_graph">
+                <img src="http://chart.apis.google.com/chart?chxr=0,0,3000&chxt=y&chs=520x140&cht=lc&chco=76A4FB,80C65A&chd=s:Tdjpsvyvttmiihgmnrst,OTbdcfhhggcTUTTUadfk&chls=2|2&chma=40,20,20,30"
+                     width="520" height="140" alt=""/>
+            </article>
+
+            <article class="stats_overview">
+                <div class="overview_today">
+                    <p class="overview_day">Today</p>
+                    <p class="overview_count">1,876</p>
+                    <p class="overview_type">Hits</p>
+                    <p class="overview_count">2,103</p>
+                    <p class="overview_type">Views</p>
+                </div>
+                <div class="overview_previous">
+                    <p class="overview_day">Yesterday</p>
+                    <p class="overview_count">1,646</p>
+                    <p class="overview_type">Hits</p>
+                    <p class="overview_count">2,054</p>
+                    <p class="overview_type">Views</p>
+                </div>
+            </article>
+            <div class="clear"></div>
+        </div>
+    </article>
+
+    <article class="module width_3_quarter">
+        <header><h3 class="tabs_involved">Content Manager</h3>
+            <ul class="tabs">
+                <li><a href="#tab1">Posts</a></li>
+                <li><a href="#tab2">Comments</a></li>
+            </ul>
+        </header>
+
+        <div class="tab_container">
+            <div id="tab1" class="tab_content">
+                <table class="tablesorter">
+                    <thead>
+                    <tr>
+                        <th></th>
+                        <th>Entry Name</th>
+                        <th>Category</th>
+                        <th>Created On</th>
+                        <th>Actions</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Lorem Ipsum Dolor Sit Amet</td>
+                        <td>Articles</td>
+                        <td>25th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Ipsum Lorem Dolor Sit Amet</td>
+                        <td>Freebies</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)